ui/src/app/workflows/components/workflows-row/workflows-row.scss

```diff
@@ -0,0 +1,22 @@
+@import 'node_modules/argo-ui/src/styles/config';
+
+.wf-rows-name {
+  line-height: 1.5em;
+  display: inline-block;
+  position: relative;
+  pointer-events: none;
+  vertical-align: middle;
+  white-space: pre-line;
+}
+.overlay {
```

Comment:
Oh, you named this `workflows-row.scss` so this is in the right place, but `.wf-rows-name` is not. and this file is imported from `workflows-row-name.tsx` instead of `workflows-row.tsx`.

if we combine those two files as I mentioned [above](https://github.com/argoproj/argo-workflows/pull/10553/files#r1461387556), then this problem just goes away

Reply:
Combined both files in #12580 to fix the mismatched import, removed the overlay and replaced with JS in #12580